app/src/main/java/com/zulip/android/MessageListFragment.java

```diff
@@ -93,16 +98,17 @@ public MessageListFragment() {
         app = ZulipApp.get();
         // Required empty public constructor
     }
-
+    List<Message> messageList;
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         if (getArguments() != null) {
             filter = getArguments().getParcelable(PARAM_FILTER);
         }
-
+        muted_messages = new ArrayList<>();
```

Comment:
Please use conventional Java variable names here. ie `mutedMessages` instead of `muted_messages`.